src/com/levi/model/Board.java

```diff
@@ -0,0 +1,90 @@
+package com.levi.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+/**
+ * Creates ships and reads locations files.
+ * @version 4.0
+ * @since 2019
+ * @author Levi Rosal
+ */
+public class Board {
+    private static final String ALPHABET = "abcdefg";
+    private static final int LINE_LIMIT = 7;
+    private static final int COLUMN_LIMIT = 7;
+    private final List<Ship> shipList = new ArrayList<>();
+    private final Player player;
+
+    public Board(final Player player) {
+        this.player = player;
+        reader(player.getLocFile());
+    }
+
+
+    /**
+     * Reads file and create ships.
+     * @param locFile Location of file
+     */
+    public void reader(final String locFile) {
+        try {
```

Comment:
Put insite the cluase try only what is need to be catched

Reply:
Done, commit 8d5d9f2.